core/math/geometry_3d.h

```diff
@@ -37,29 +37,24 @@
 
 class Geometry3D {
 public:
-	static void get_closest_points_between_segments(const Vector3 &p1, const Vector3 &p2, const Vector3 &q1, const Vector3 &q2, Vector3 &c1, Vector3 &c2) {
-// Do the function 'd' as defined by pb. I think it's a dot product of some sort.
-#define d_of(m, n, o, p) ((m.x - n.x) * (o.x - p.x) + (m.y - n.y) * (o.y - p.y) + (m.z - n.z) * (o.z - p.z))
+	static void get_closest_points_between_segments(const Vector3 &a1, const Vector3 &a2, const Vector3 &b1, const Vector3 &b2, Vector3 &c1, Vector3 &c2) {
```

Comment:
Godot uses `p_` syntax for passed in parameters and `r_` syntax for ones that modify the paramters.